mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { FC, useEffect } from 'react';
```

Comment:
Component name, file name, folder name should be the same, let's use uam-configure-group